src/main/java/longtimenosee/model/event/Description.java

```diff
@@ -9,13 +9,13 @@
  */
 public class Description {
 
-
+    public static final int MAXIMUM_DESCRIPTION_LENGTH = 175;
     public static final String MESSAGE_CONSTRAINTS = "Description should only contain "
-        + "alphanumeric characters and spaces.";
+        + "alphanumeric characters and spaces. It should not be empty";
+    public static final String LENGTH_CONSTRAINTS = "Ensure description is of an appropriate length";
```

Comment:
Maybe it would be clearer to the user if we attach the MAXIMUM_DESCRIPTION_LENGTH to the LENGTH_CONSTRAINTS message. Something like  "Ensure description is of an appropriate length of: " MAXIMUM_DESCRITION LENGTH + " characters."